### PR TITLE
Fix some links found with linkchecker

### PR DESCRIPTION
**Pull request by SvenDowideit**
Signed-off-by: Sven Dowideit <SvenDowideit@home.org.au>

### CODE REVIEW

File: docs/install-machine.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ installation</a> instructions.
 
 If you only want Docker Machine, you can install the Machine binaries (the
 latest versions of which are located at
-https://github.com/docker/machine/releases/) directly by following the
+https://github.com/docker/machine/releases/ ) directly by following the
```
Comment (dmp42) [inline docs/install-machine.md]: Not sure why we need this? https://github.com/docker/machine/blob/master/docs/install-machine.md looks ok

File: docs/reference/create.md [hunk 2]
```diff
@@ -126,7 +126,7 @@ Labels:
 The supported flags are as follows:
 
 - `--engine-insecure-registry`: Specify [insecure registries](https://docs.docker.com/reference/commandline/cli/#insecure-registries) to allow with the created engine
-- `--engine-registry-mirror`: Specify <a href="https://github.com/docker/docker/blob/master/docs/sources/articles/registry_mirror.md">registry mirrors</a> to use
+- `--engine-registry-mirror`: Specify <a href="https://github.com/docker/docker/blob/master/docs/articles/registry_mirror.md">registry mirrors</a> to use
```
Comment (dmp42) [inline docs/reference/create.md]: This doc is empty / confusing.
We should rather link to https://github.com/docker/distribution/blob/master/docs/mirror.md

File: docs/install-machine.md (hunk 1 above)
Comment (nathanleclaire) [inline docs/install-machine.md]: @SvenDowideit does it confuse the markdown parsing to have another way?

Comment (SvenDowideit) [inline docs/install-machine.md]: yup. you get an invalid link with an encoded `)`.

**Comment by nathanleclaire** [comment]
Can you update to @dmp42 suggestion as well please?  The link in the doc points to a "deprecated" / empty doc.  I know it was like that before but might as well change while we're there :)  Thanks!

File: docs/reference/create.md (hunk 2 above)
Comment (SvenDowideit) [inline docs/reference/create.md]: agreed - though I was going for minimally invasive, updating.

**Comment by SvenDowideit** [comment]
@nathanleclaire sold :)

**Comment by nathanleclaire** [comment]
:moneybag:

**Comment by nathanleclaire** [comment]
LGTM

**Pull request merged**